### DISCUSSION TITLE
Staking instruction

### DISCUSSION THREAD

**Discussion by astede** [iotaledger/firefly#7439] [Features / Enhancements]
The instruction for staking states "Open network configuration in the advanced settings and make sure the current node has the "Participation" feature by clicking "View info."

When I go to Advanced, Network Config, all I see under the Node selection is either Automatic or Manual selection. Should the current  statement in Iota be changed to reflect the correct wording, or am I missing something here?

**Comment by begonaalvarezd** [comment]
Hi, perhaps it can be improved but the idea is that on manual you can see the node list and click on the "..." menu to see more info 
![image](https://github.com/iotaledger/firefly/assets/3624944/fd8580fb-8621-4446-9c23-72c0ad6b1519)
![image](https://github.com/iotaledger/firefly/assets/3624944/82aab6d1-7caf-4aec-af92-05f1be91012d)

**Reply by astede** [reply]
Thank you for your explanation. I clicked on manual and found that the default was iota....cafe, so I changed the default to iota.org.
However, I switched back to the automatic setting. Does that mean that when the next round no 7? begins, I will receive an automatic message, irrespective of what the manual setting is set to?

**Reply by begonaalvarezd** [reply]
The official node list always has the staking events, so no need to update to manual or automatic. About the staking rounds, I recommend reading the latest announcement from the IOTA Foundation https://blog.iota.org/stardust-upgrade-iota-tokenomics/ in summary, there will be no more staking rounds but on the upcoming iota network upgrade you will get an airdrop in IOTAs to compensate your staked rewards

**Comment by astede** [comment]
I appreciate your time and explanation and thank you for your assistance. Very useful information!
Kind Regards
Astede
